moveitpy_simple/moveitpy/__init__.py

```diff
@@ -494,6 +494,145 @@ def ik(self, pose: list[float], link_name: str) -> list[float] | None:
         return None
 
 
+class FullRobot(RobotComponent):
+    """A class that wraps around a combined panda and hand joint model group & planning component."""
+
+    def __init__(  # noqa: PLR0913
+        self,
+        robot_model: RobotModel,
+        full_robot_joint_model_group: JointModelGroup,
+        full_robot_planning_component: PlanningComponent,
+        full_robot_planning_scene_monitor: PlanningSceneMonitor,
+        arm_component: RobotComponent,
+        gripper_component: RobotComponent,
```

Comment:
Why do we need these two?

Reply:
to get joint limits for arm and gripper separately. we redefine gripper open close states in urdf separately under gripper group. so not sure planning component can access subgroups. I tried directing reading from franka urdf, but there the gripper limit is 0.04.